### ISSUE TITLE
Session timed out, despite session_timeout limit not reached

### ISSUE THREAD

**Issue by slussinator** [ctreffe/alfred#243]
Dear Community, 

I just received feedback from a participant that tried to take part in one of my studies yesterday. In the middle of the experiment they received the following message "Session expired. Sorry! the session timed out." (See attached file). The session timeout was set at 30 minutes, so this should not be the culprit. Do you have an idea what caused the problem? I remember eduroam not working well for a short period yesterday. Could this have caused the problem? 
[Session expired.pdf](https://github.com/ctreffe/alfred/files/11623394/Session.expired.pdf)

Thank you in advance!

**Comment by ctreffe** [comment]
Hey @slussinator,

your issue concerns an online experiment, correct? In this case, I see no reason why the short sesseion timeout should _not_ be to blame for the aborted session.

Either way, you can easily check whether your experiment sessions run according to the configured timeout by running a few test sessions and waiting between page changes. At this point, from a technical perspective, I see no technical issues or alternative explanations for the aborted session. Also, I see no way that temporary problems with an internet connection could cause the described problem.

We will, of course, revisit this issue, if you encounter any verifiable problems with the timeout configuration.

**Comment by slussinator** [comment]
ok. I guess maybe they just took longer than expected? Does the session timeout relate to pages or sections? Possibly it took more than half an hour to move through the section and this is why it timed out? I do understand correct though that the session timeout does not apply to the time needed to finish the whole experiment, correct? 
Thanks in advance!

**Comment by ctreffe** [comment]
This is indeed unclear from the documentation, or at least I had the same impression as @slussinator since in older versions of Alfred, the session timeout worked as she described. We should add an explanation with an emphasis on the fact that the timeout works with regard to the duration of the ongoing session.

**Comment by jobrachem** [comment]
I agree, clarifying the documentation in this point would definitely be valuable. Thinking further, it may also be sensible to change the behavior of the session timeout, such that a session times out with respect to the most recent activity instead of the first activity, as you expected it to. From the top of my head, I do not see any definite reason that speaks against such a change.
